crates/gitbutler-repo/src/rebase.rs

```diff
@@ -355,621 +420,23 @@ impl ConflictEntries {
 
         set.len()
     }
-}
-
-/// Automatically resolves an index with a preferences for the "our" side
-///
-/// Within our rebasing and merging logic, "their" is the commit that is getting
-/// cherry picked, and "our" is the commit that it is getting cherry picked on
-/// to.
-///
-/// This means that if we experience a conflict, we drop the changes that are
-/// in the commit that is getting cherry picked in favor of what came before it
-fn resolve_index(
-    repository: &git2::Repository,
-    index: &mut git2::Index,
-) -> Result<ConflictEntries, anyhow::Error> {
```

Comment:
It does make me happy that we no longer take care of this concern

Reply:
While doing `gix merge` index creation I realized that the index file as we know it is utterly underdefined for the complexities of merge conflicts - it simply can't represent them properly. In the past, when trivial merges without rename tracking were done, that wasn't the case, but now… it's a lossy science and there is definitely room to innovate here.

Reply:
That is really cool :D